packages/consumption/test/modules/requests/DecideRequestParametersValidator.test.ts

```diff
@@ -315,8 +315,10 @@ describe("DecideRequestParametersValidator", function () {
             return;
         }
 
-        expect(validationResult.error.code).toBe("inheritedFromItem");
-        expect(validationResult.error.message).toBe("Some child items have errors.");
+        expect(validationResult.error.code).toBe("error.consumption.validation.inheritedFromItem");
+        expect(validationResult.error.message).toBe(
+            "Some child items have errors. If this error occurred during the specification of a Request, call 'canCreate' to get more information."
+        );
```

Comment:
we have a helper method `expect(result).errorValidationResult(...)`, please use that.